packages/web-components/fast-foundation/src/picker/picker-list-item.ts

```diff
@@ -29,6 +29,17 @@ export class FASTPickerListItem extends FASTElement {
     @attr({ attribute: "value" })
     public value: string;
 
+    /**
+     * Disables the picker-list-item.
+     *
+     * @public
+     * @remarks
+     * HTML Attribute: disabled
+     */
+    @attr({ attribute: "disabled", mode: "boolean" })
```

Comment:
```suggestion
    @attr({ mode: "boolean" })
```

I don't think you need to specify the attribute name since it's the same as the property name.